src/qos/logger.js

```diff
@@ -28,7 +28,7 @@ class Logger {
     }
 
     if (group !== 'default') {
-      message = group + ': ' + message
+      message = '[' + Game.shard.name + ']' + group + ': ' + message
```

Comment:
This would look cleaner using template literals. There should also be a space between that bracket and the group.